sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -91,6 +92,58 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
     return query.getOne();
   }
 
+  async getPendingAssessment(
```

Comment:
Are we not selecting for institution location with integration as true?

Also, is an IER 12 a location wise file or one single file for all institution locations of all institutions?

Reply:
You are right @dheepak-aot, i ve changed it now. Thanks for noting.

Reply:
What about location? is it not a location wise file?